uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/app.js

```diff
@@ -0,0 +1,28 @@
+define([
+  'common/angular',
+  'common/angularBootstrap',
+  'common/services/angular-date/date-format',
+  'leave-absences/shared/ui-router',
+  'leave-absences/absence-tab/modules/settings',
```

Comment:
you are missing a `modules/config`, that in turn requires `modules/settings`. please see the other angular apps in L&A